src/site/markdown/java-api.md

```diff
@@ -0,0 +1,564 @@
+## Java API
+
+Now that you know how to configure MyBatis and create mappings, you're ready for the good stuff. The MyBatis Java API is where you get to reap the rewards of your efforts. As you'll see, compared to JDBC, MyBatis greatly simplifies your code and keeps it clean, easy to understand and maintain. MyBatis 3 has introduced a number of significant improvements to make working with SQL Maps even better.
+
+### Directory Structure
+
+Before we dive in to the Java API itself, it's important to understand the best practices surrounding directory structures. MyBatis is very flexible, and you can do almost anything with your files. But as with any framework, there's a preferred way.
+
+Let's look at a typical application directory structure:
+
+```
```

Comment:
The block displays some highlights which we don't need
 ![dir](https://user-images.githubusercontent.com/69554233/217540826-68bead29-26aa-43bf-b9fd-3af79d29496e.png)
 @harawata

Reply:
Thank you, @awxiaoxian2020 !

This seems like a bug (in Doxia, probably?) to me.
Although it is annoying, it is not a deal-breaker, so let's just leave it for now.

This is the only remaining issue and the PR is ready for review, is that right?

Reply:
I will use HTML Tidy to check it again.

Reply:
Roger 👍 
Please take your time and let me know when it's ready (or use the draft status).

Reply:
> Roger 👍 Please take your time and let me know when it's ready (or use the draft status).

Actually, you can review it now if you have spare time, I just do a final check. :)